### DISCUSSION TITLE
Configuration with GOOGLE WORKSPACE

### DISCUSSION THREAD

**Discussion by Meridiongroup** [YetiForceCompany/YetiForceCRM#16427] [General]
Is there a way to make the "my mail" module work with google workspace?
Do I have problems with just that? Can someone help me?
Thank you

**Comment by mariuszkrzaczkowski** [comment]
> Is there a way to make the "my mail" module work with google workspace? 

Clarify what you mean?

**Reply by Meridiongroup** [reply]
![RoundCube](https://user-images.githubusercontent.com/112404611/187659969-7b993f40-cd11-4a7f-99dd-7ee42a42a6d7.JPG)
This is my configuration

![Mymail](https://user-images.githubusercontent.com/112404611/187660070-4ae57e28-0f04-48d9-a985-a7ac8eb3df0b.JPG)
But when i try to log in, i get error

**Reply by mariuszkrzaczkowski** [reply]
logs

**Reply by Meridiongroup** [reply]
Where do I find the logs?

**Reply by mariuszkrzaczkowski** [reply]
https://yetiforce.com/en/knowledge-base/documentation/developer-documentation/item/debugging

**Reply by Meridiongroup** [reply]
31-Aug-2022 06:36:16 -0400]: IMAP Error: Login failed for [informazioni.sii@gmail.com](mailto:informazioni.sii@gmail.com) against imap.gmail.com from *** AUTHENTICATE PLAIN: Invalid credentials (Failure) in ***/PINTO/public_html/modules/OSSMail/roundcube/program/lib/Roundcube/rcube_imap.php on line 211 (POST /PINTO/public_html/modules/OSSMail/roundcube/?_task=login&_action=login)

This is the log , but user and password are both correct

**Reply by Meridiongroup** [reply]
Update?
Thanks

**Reply by rskrzypczak** [reply]
There is nothing else to add here. 
You have the cause and the solution for your problem described here: https://github.com/YetiForceCompany/YetiForceCRM/discussions/16427#discussioncomment-3532556

**Comment by rskrzypczak** [comment]
Here is the solution to your problem: https://support.google.com/accounts/thread/166395190/keep-getting-authenticationfailed-invalid-credentials-failure-error-in-outlook?hl=en
There is no YetiForce error here.
